### PR TITLE
new UI retro components

### PR DESCRIPTION
**Pull request by edderrd**
-  updated storybook
- added retro button
- added toolbar
- added dialog
- added geneva and chicagoflf
- added some icons

## Screenshots

Screen Shot 2021-12-07 at 11 25 26 PM
Screen Shot 2021-12-07 at 11 25 53 PM
Screen Shot 2021-12-07 at 11 25 57 PM
Screen Shot 2021-12-07 at 11 30 51 PM

### CODE REVIEW

File: components/ui/icons/index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import withSvg from '../../hoc/withSvg';
+
+import LevelLogoIcon from './LevelLogo.svg';
```
Comment (gustavoalfaro) [inline components/ui/icons/index.js:3]: We may want to do this like `LevelLogoSvg` and export the icon as `LevelLogoIcon`

**Review by gustavoalfaro — COMMENTED** [review]
Nice approach,  just some suggestions, and we may need to standardize from the beginning by adding a linter before adding much more code, and also a good idea would be using `rem` instead of pixels or any other relative measurement

**Comment by mckelvey** [comment]
Re: Linting: I’ve done a bunch of the base linting/updates to the Vercel boilerplate of late; I’ll make a PR to bring that in here.

File: .storybook/preview-head.html [hunk 2]
```diff
@@ -2,6 +2,7 @@
   href="https://fonts.googleapis.com/css2?family=Work+Sans:wght@400;500;600&display=swap"
   rel="stylesheet"
 ></link>
+<link rel="stylesheet" href="https://fontlibrary.org//face/chicagoflf" type="text/css"></link>
```
Comment (mckelvey) [inline .storybook/preview-head.html:5]: `https://fontlibrary.org/face/chicagoflf` (double slash before face)

File: pages/_document.js [hunk 3]
```diff
@@ -39,6 +39,7 @@ export default class MyDocument extends Document {
             href="https://fonts.googleapis.com/css2?family=Work+Sans:wght@400;500;600&display=swap"
             rel="stylesheet"
           ></link>
+          <link rel="stylesheet" href="https://fontlibrary.org//face/chicagoflf" type="text/css"></link>
```
Comment (mckelvey) [inline pages/_document.js:42]: Same note on URL here as above.

**Review by mckelvey — COMMENTED** [review]
@edderrd — What a great start! I had just a URL note. Don’t worry about linting as I’ll follow with that.

File: components/hoc/withSvg.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
+import styled, { css } from 'styled-components';
+import PropTypes from 'prop-types';
+
+const withSvg = (icon, width, height) => {
+  const SvgIcon = styled(icon)`
+    ${({
+      override: {
+        width: overrideWidth = width,
```
[the remark this answers is not in the document]
Comment (edderrd) [reply components/hoc/withSvg.js:8]: that's a fallback value in case there's no override defined

**Comment by edderrd** [comment]
@mckelvey changes made

**Review by mckelvey — APPROVED** [review]
Nice work @edderrd !

**Pull request merged**